source/release-notes/5.0.txt

```diff
@@ -186,15 +195,45 @@ for the ``"matchPrimaryNode"`` option. When set for a
 :binary:`~bin.mongod`, the instance follows the behavior specified for
 the ``"disabled"`` option.
 
-.. _5.0-rel-notes-security:
+.. _5.0-rel-notes-shell:
 
-Security
---------
+Shell Changes
+-------------
 
-MongoDB 5.0 introduces the :parameter:`opensslCipherSuiteConfig`
-parameter to enable configuration of the supported cipher suites OpenSSL
-should permit when using TLS 1.3 encryption.
+New ``mongosh`` shell
+~~~~~~~~~~~~~~~~~~~~~
+
+The ``mongo`` shell has been deprecated in MongoDB v5.0. The
+replacement shell is :mongosh:`mongosh </>`. The legacy ``mongo`` shell
+will be removed in a future release.
+
+The ``monogsh`` shell is not packaged with MongoDB. See the 
```

Comment:
```suggestion
The ``mongosh`` shell is not packaged with MongoDB. See the 
```

Reply:
Aside from the typo, same considerations as above on the shell being packaged with MongoDB.